modules/docs/arrow-docs/docs/_data/commons.yml

```diff
@@ -2,5 +2,5 @@ name: "Arrow"
 title: "Λrrow"
 description: "Functional companion to Kotlin's Standard Library"
 keywords: "functional-programming, kotlin-library, monads, monad-transformers, functional-data-structure, kotlin, fp-types, adt, free-monads, tagless-final, mtl, for-comprehension, category-theory"
-google_analytics: "UA-110153729-1"
+google_analytics: UA-18433785-18
```

Comment:
Shouldn't this be a string?

Reply:
It is a string, but YAML string syntax is a bit all over the place, because in general you don't need quotes unless you want to force convert something that doesn't look like a string (e.g. a number: `10`).

But then you can read different opinions on how correct should be to always use quotation marks, or not at all. Anyway, probably, for the sake of consistency with the rest of the file this should be in quotation marks, really.

I can correct it in a future PR.